packages/shared/src/components/post/collection/CollectionPostContent.tsx

```diff
@@ -0,0 +1,160 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { LazyImage } from '../../LazyImage';
+import {
+  ToastSubject,
+  useToastNotification,
+} from '../../../hooks/useToastNotification';
+import PostContentContainer from '../PostContentContainer';
+import usePostContent from '../../../hooks/usePostContent';
+import FixedPostNavigation from '../FixedPostNavigation';
+import { BasePostContent } from '../BasePostContent';
+import { cloudinary } from '../../../lib/image';
+import { Separator } from '../../cards/common';
+import { postDateFormat } from '../../../lib/dateFormat';
+import Markdown from '../../Markdown';
+import CollectionPostWidgets from './CollectionPostWidgets';
+import CollectionPostHeaderActions from './CollectionPostHeaderActions';
+import {
+  PostContainer,
+  PostContentProps,
+  PostNavigationProps,
+} from '../common';
+import Pill from '../../Pill';
+import { CollectionsIntro } from '../widgets';
+
+const CollectionPostContent = ({
```

Comment:
Which one is used now, I see changes in the ArticlePostModal, but then also this dedicated component.
Not sure I'm following the change maybe?

Reply:
Yeah, it did not follow initial DR plan, I am working on moving everything to separate one.